skore/src/skore/sklearn/_estimator/feature_importance_accessor.py

```diff
@@ -203,10 +203,24 @@ def coefficients(self) -> pd.DataFrame:
             if isinstance(parent_estimator, Pipeline)
             else parent_estimator
         )
-        intercept = np.atleast_2d(estimator.intercept_)
+        try:
+            intercept = np.atleast_2d(estimator.intercept_)
+        except AttributeError as msg:
+            if "object has no attribute 'intercept_'" in str(
+                msg
+            ):  # for SGDOneClassSVM()
+                intercept = None
+            else:
+                raise
```

Comment:
I think we can simplify because there is no other potential `AttributeError` to be raised by this code.

```suggestion
        try:
            intercept = np.atleast_2d(estimator.intercept_)
        except AttributeError:
            # SGDOneClassSVM does not expose an intercept
            intercept = None
```